jobserver/urls.py

```diff
@@ -273,9 +273,8 @@
     ),
     path("jobs/<identifier>/", JobDetailRedirect.as_view(), name="job-redirect"),
     path("login/", Login.as_view(), name="login"),
-    path(
-        "login-with-token/<str:token>/", LoginWithURL.as_view(), name="login-with-url"
-    ),
+    path("login-with-url/<str:token>/", LoginWithURL.as_view(), name="login-with-url"),
```

Comment:
Note - I renamed this url, which means any emails generated before we deploy won't work. But they only last an hour anyway...